util/resources.go

```diff
@@ -35,6 +35,8 @@ type Resources struct {
 	// Memory supports values up to Terrabytes (tb). If the unit is ommited, then it
 	// is assumed to be bytes. This is not case sensitive.
 	Memory string `json:"memory"`
+	// Volumes to be used by each node.
+	Volumes *[]string `json:"volumes"`
```

Comment:
Resource owns Volumes, so it should be []string. Should only be a pointer if it is an immovable type such as a mutex or if it is a reference to data owned by another.

Reply:
OK I see.

Reply:
Fixed now.